### ISSUE TITLE
Change list of pre-installed tracks/datasets

### ISSUE THREAD

**Issue by danledinh** [hms-dbmi/higlass-docker#144]
Hi, is there a way to remove some pre-installed tracks/datasets? 

I'm currently using the Django admin page to manage my own data, but I'd like to make some changes to the default list.

Thanks!

**Comment by pkerpedjiev** [comment]
Yup. In the django admin interface, under the viewconfs table you should find an entry for 'default'. In that viewconf, you'll find a list called `trackSourceServers`. If you take out `higlass.io` it should remove the default list.

**Comment by danledinh** [comment]
Cool. Is there a way to select a subset of the data? I.e. I'd like to keep the hg38 chromosome annotations.

**Comment by pkerpedjiev** [comment]
Unfortunately right now there isn't. Sorry about that.

**Comment by danledinh** [comment]
ok. Thanks!

**Comment by danledinh** [comment]
I was able to create and ingest gene annotation files. What kind of file is necessary to correctly display the position search box? 

The screenshot shows my position search box but the coordinates are fixed:
![image](https://user-images.githubusercontent.com/12414317/34584251-d956a2b2-f14f-11e7-9d74-73ec7af5654a.png)

**Comment by pkerpedjiev** [comment]
You need to upload a chromSizes file and set it's `coordSystem` parameter.

```
docker exec higlass-container python higlass-server/manage.py \
  ingest_tileset \
  --filename /chromSizes.tsv \
  --datatype chromsizes \
  --filetype chromsizes-tsv \
  --coordSystem myAssembly
```

If you want gene autocomplete as well, you'll need a gene-annotations beddb file with the same coordSystem as the chromSizes file.

**Comment by danledinh** [comment]
That did the trick! Thanks again!!!
